feat(content): Add a short mission about an Argosy hijacking in Syndicate space (#8479)
endless-sky/endless-sky: data/human/human missions.txt

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

mission "Argosy Hijacking (Trigger)"
	invisible
	landing
	source
		government "Syndicate"
	to offer
		random < 10
	on offer
		fail

mission "Argosy Hijacking"
	name "Stop ship hijacking"
	description "The <npc> was hijacked on <origin>; disable it and return to <origin>. Do not capture or destroy it; Syndicated Security will deal with the hijackers in space after you have disabled it."
	landing
	source
		government "Syndicate"
		not attributes "station"
	to offer
		has "Argosy Hijacking (Trigger): offered"
	on offer
		conversation

> 			`You're signing the landing papers with the Syndicated Security officer on <origin>, a standard procedure after landing in a planet within the Syndicate bloc. "Everything seems to be in order. And of course, we'll have your ship refuelled promptly. Have a good day, Captain <last>." He nods his head at you and starts to move off towards the next ship.`

Reviewer: Works great for me! You can resolve the convo, if you wish 👍

			`	Just before he is able to, however, you hear a commotion going on by the <npc model> on the landing pad next to you.`
			choice
				`	(Turn around to see what's going on.)`
					goto turn
				`	(Ignore the commotion.)`
					goto ignore
			label turn
			`	Both of you turn your head to see what's happening, but as you do, there's a deafening blare of warning sirens through <origin>'s spaceport. Suddenly, the <npc model> on the landing pad behind you, the <npc>, jolts up with its loading ramp still extended.`
				goto hijack
			
			label ignore
			`	Figuring it's nothing, you turn back towards the <ship> only to hear a deafening blare of warning sirens through <origin>'s spaceport. Looking around instinctively, you notice from the corner of your eye that the <npc>, that <npc model> on the landing pad next to you, jolts up with its loading ramp still extended.`

			label hijack

> 			`	"Hey, what's going on here?" the officer tries to shout over the noise of the sirens and the repulsors, but the rogue <npc model> veers rapidly and uncertainly towards his direction. He barely jumps down for cover before the ship comes up towards him to clip a stack of cargo crates, sending it flying into the refuelling platform. With its repulsors at full power, it narrowly avoids colliding with the <ship> and flees away into the sky.`

Reviewer 2: ```suggestion
			`	"Hey, what's going on here?" the officer tries to shout over the noise of the sirens and the repulsors, but the rogue <npc model> veers rapidly and uncertainly towards his direction. He barely jumps down for cover before the ship comes up towards him to clip a stack of cargo crates, sending them flying into the refueling platform. With its repulsors at full power, it narrowly avoids colliding with the <ship> and flees away into the sky.`
```

			choice
				`	"Well, yeah."`
				`	"No, that's not what I meant."`
			`	He scoffs at you, and lets you go with a rough push. "I don't need your explanation, I don't think you could give me one. I just can't understand how you can walk away feeling alright that you've set back one man's life so much."`
				goto end
			
			label insurance
			`	He looks off to his left for just a moment with a look of loss in his eyes, then lets you go with a rough push. "Mmm, no, like a cocksure bastard I sure didn't. Ah, now that one isn't your fault, but if you had just taken your finger off the trigger for a few seconds less, I wouldn't be in this position now, would I? Bastards, you and the hijackers..."`

			label end
			`Suddenly, he turns around and walks off towards the direction of the bar, leaving you to contemplate your actions.`

	on complete
		payment 28000
		conversation 
			`As you land, Syndicated Security have the <npc> in tow, escorted by a trio of Quicksilvers. The formation lands on one of the larger pads reserved for Syndicated Security vessels, where it is fenced off from the rest of the spaceport.`

> 			`	The hijackers, just a raucous group of young kids, are hauled out of the <npc model> by Syndicated Security personnel towards a security van. Multiple times the kids try to run away, but after a few minutes they're finally detained; the van drives off towards the spaceport terminal.`

Author: I'll change it up a little: 

```suggestion
			`	The hijackers, just a raucous group of young kids, are hauled out of the <npc model> by Syndicated Security personnel towards a security van. Multiple times the kids try to run away, but after a few minutes they're finally detained; the van finally drives off towards the spaceport terminal.`
```

	on offer
		event "argosy hijacking trigger"
		fail

event "argosy hijacking trigger"

mission "Argosy Hijacking"
	landing
	name "Stop Ship Hijacking"
	description "The <npc> was hijacked on <origin>; disable it and return to <origin>. Do not capture or destroy it; Syndicated Security will deal with the hijackers in space after you have disabled it."
	source
		government "Syndicate"
		not attributes "station"
		near "Alcyone" 4
		near "Almach" 3
		near "Gienah" 3
	to offer
		has "event: argosy hijacking trigger"
	substitutions
		"<argosy hijacking variable>" "destroy"
		"<argosy hijacking variable>" "board"
			has "boarded hijacked argosy"
	on offer
		conversation
			`You're signing the standard Syndicate landing papers with an officer on <origin>. "Everything seems to be in order. And of course, we'll have your ship refueled promptly. Have a good day, Captain <last>." He nods his head at you and starts to move off towards the next ship.`
			`	Before he can make it there, you hear a commotion by the <npc model> on the landing pad next to you.`
			choice
				`	(Turn around to see what's going on.)`
					goto turn
				`	(Ignore the commotion.)`
					goto ignore
			label turn

> 			`	Both of you turn your head to see what's happening, but as you do a deafening blare of warning sounds through <origin>'s spaceport. Suddenly, the <npc model> on the landing pad behind you, the <npc>, jolts up with its loading ramp still extended.`

Reviewer 3: ```
			`	Suddenly, the <npc model> on the landing pad behind you, the <npc>, jolts up with its loading ramp still extended.`
```
I feel like this could be rephrased so there's less comma. It's a little jarring to read.

		dialog `The <npc> has been disabled. Syndicated Security will now deal with the hijackers, but you should go back and report to the captain whose ship was hijacked.`
		on board
			set "boarded hijacked argosy"
			dialog `You have failed this mission because you boarded the ship, against the orders of the Syndicated Security officer. The captain whose ship was hijacked won't be impressed.`
			fail

	on fail
		conversation
			`When you land, the captain of the <npc> comes up to you, his face rather red.`
			`	He grabs your shoulders and shakes you hard, shouting, "Didn't I tell you not to <argosy hijacking variable> my ship? Do you know how many credits I've just lost because of you?"`
			choice
				`	"I'm sorry, I was just trying to help!"`
				`	"I'm sure whatever you lost wasn't worth all that much."`
					goto worthless
			`	He stops shaking you for a moment to say, "Well, you shouldn't have blown up my <npc model> while you were trying to help, weren't you? Fat lot of help that was, even if you tried to do the right thing. Oh, well..."`
			`	He lets you go with a push and turns around to walk off towards the direction of the bar, leaving you to contemplate your actions.`

> 

Reviewer 3: Missing end node or `goto`.

			

event "quicksilver mixup 2 available"

mission "Argosy Hijacking (Trigger)"
	invisible
	source
		government "Syndicate"
	to offer
		random < 10
	on offer
		event "argosy hijacking trigger" 14 35
		fail

event "argosy hijacking trigger"

mission "Argosy Hijacking"
	name "Stop ship hijacking"
	description "The <npc> was hijacked on <origin>; disable it and return to <origin>. Do not capture or destroy it; Syndicated Security will deal with the hijackers in space after you have disabled it."
	landing
	source
		government "Syndicate"
		not attributes "station"
		near "Alcyone" 4
		near "Almach" 3

> 		near "Gienah" 3

Reviewer 4: Shouldn't these be under an `or` block?

Reviewer 5: `or` isn't a thing inside location filters.

> 		near "Alcyone" 4
> 		near "Almach" 3
> 		near "Gienah" 3

Reviewer 6: ```suggestion
		not near "Sol" 3
```
Since warp_core mentioned how we can't use `or` for location filters, the next closest thing is probably having it be a Syndicate world away from Sol. Not an exact fit, but it's close.

	to offer
		has "Argosy Hijacking (Trigger): offered"
		has "event: argosy hijacking trigger"
	on offer
		conversation

> 			`You're signing the landing papers with the Syndicated Security officer on <origin>, a standard procedure after landing in a planet within the Syndicate bloc. "Everything seems to be in order. And of course, we'll have your ship refuelled promptly. Have a good day, Captain <last>." He nods his head at you and starts to move off towards the next ship.`

Reviewer 4: Adjusting sentence flow.
```suggestion
			`You're signing the standard Syndicate landing papers with an officer on <origin>. "Everything seems to be in order. And of course, we'll have your ship refueled promptly. Have a good day, Captain <last>." He nods his head at you and starts to move off towards the next ship.`
```

			`	Just before he is able to, however, you hear a commotion going on by the <npc model> on the landing pad next to you.`
			choice
				`	(Turn around to see what's going on.)`
					goto turn
				`	(Ignore the commotion.)`
					goto ignore
			label turn

> 			`	Both of you turn your head to see what's happening, but as you do, there's a deafening blare of warning sirens through <origin>'s spaceport. Suddenly, the <npc model> on the landing pad behind you, the <npc>, jolts up with its loading ramp still extended.`

Reviewer 4: ```suggestion
			`	Both of you turn your head to see what's happening, but as you do a deafening blare of warning sounds through <origin>'s spaceport. Suddenly, the <npc model> on the landing pad behind you, the <npc>, jolts up with its loading ramp still extended.`
```

				goto hijack
			
			label ignore

> 			`	Figuring it's nothing, you turn back towards the <ship> only to hear a deafening blare of warning sirens through <origin>'s spaceport. Looking around instinctively, you notice from the corner of your eye that the <npc>, that <npc model> on the landing pad next to you, jolts up with its loading ramp still extended.`

Reviewer 4: "Sirens" seems unnecessary to me.
```suggestion
			`	You turn back towards the <ship> only to hear a deafening blare of warning through <origin>'s spaceport. Looking around instinctively, you notice from the corner of your eye that the <npc>, that <npc model> on the landing pad next to you, jolts up with its loading ramp still extended.`
```

Author: How about:
```suggestion
			`	You turn back towards the <ship> only to hear a deafening blare of warning sounds through <origin>'s spaceport. Looking around instinctively, you notice from the corner of your eye that the <npc>, that <npc model> on the landing pad next to you, jolts up with its loading ramp still extended.`
```

Reviewer 4: Are you using sounds as a verb or noun here?

Author: Noun.

Author: Okay, I see how you might have misread this. I'm changing it back to sirens, though.

			label hijack

> 			`	"Hey, what's going on here?" the officer tries to shout over the noise of the sirens and the repulsors, but the rogue <npc model> veers rapidly and uncertainly towards his direction and the <ship>. He barely jumps down for cover before the ship comes up towards him to clip a stack of cargo crates, sending them flying into the refueling platform. With its repulsors at full power, it narrowly avoids colliding with the <ship> and flees away into the sky.`

Reviewer 4: ```suggestion
			`	"Hey, what's going on here?" the officer tries to shout over the noise of the sirens and the repulsors, but the rogue <npc model> veers rapidly and uncertainly towards his direction and the <ship>. He can barely jump down for cover before the ship clips a stack of cargo crates, sending them flying onto the refueling platform. With its repulsors at full power, it narrowly avoids colliding with the <ship> and flees away into the sky.`
```

Author: No, I think the cargo crates would fly into the platform, not onto it; it sounds like they've landed on the platform instead of colliding with it.
```suggestion
			`	"Hey, what's going on here?" the officer tries to shout over the noise of the sirens and the repulsors, but the rogue <npc model> veers rapidly and uncertainly towards his direction and the <ship>. He can barely jump down for cover before the ship clips a stack of cargo crates, sending them flying into the refueling platform. With its repulsors at full power, it narrowly avoids colliding with the <ship> and flees away into the sky.`
```

			`	The Syndicated Security officer, dazed and coated in dust kicked up from the unpermitted departure, only just pulls himself up before another person runs up to accost him. "Oi, those kids, they just hijacked my <npc model>! Get someone on their tail!"`
			choice
				`	"I'll do it!"`
				`	(Let Syndicated Security deal with it.)`
					goto securityjob

> 			action
> 				set "argosy hijacking accepted"

Reviewer 4: ```suggestion
```

> 			`	The captain turns towards you. "Thank you, Captain! Don't destroy the <npc>, I'm already late for my bloody contract. Agh, this one doesn't help at all," he mutters to himself.`

Reviewer 4: What's the second dialogue sentence mean?

Author: I'm not sure what you mean.

Reviewer 4: What's he referring to when he says, "this one doesn't help at all"? What's the "one"? Are there more like it?

Author: He's already late, so he's been held back some time before for this contract.

I could just remove 'one'.

			`	The officer looks back at you, having dusted himself off and regained his senses. "Yes, go and deal with his ship now. We'll have Syndicated Security ships come after you to arrest the hijackers and tug it down. Go!"`
				launch
			
			label securityjob
			`	The officer, slightly flustered, calls for a small squadron of Syndicated Security ships to chase the hijacked <npc>, then runs off to one of <origin>'s spaceport terminals to file the paperwork with the captain.`
				decline
	

> 	npc save disable

Author: Yes, I didn't think of that.

		personality launching daring uninterested target
			confusion 40
		government "Bounty (Disguised)"
		ship "Argosy (Blaster)" "Moonshake Express"
		dialog `You've disabled the <npc>. Syndicated Security will now deal with the hijackers, but you should go back and report to the captain whose ship was hijacked.`

> 		to spawn
> 			has "argosy hijacking accepted"

Reviewer 4: If the job is declined, they won't spawn anyway.
```suggestion
```

			set "boarded hijacked argosy"
			dialog `You have failed this mission because you boarded the ship, against the orders of the Syndicated Security officer. The captain whose ship was hijacked won't be impressed.`
			fail

	on fail
		conversation
			`When you land, the captain of the <npc> comes up to you, his face rather red.`

> 			`	He grabs your shoulders and shakes you hard, shouting, "Didn't I tell you not to <argosy hijacking variable> my ship? Do you know how many credits I've just lost because of you?"`

Reviewer 6: It looks like substitutions are set in stone once the mission is accepted, so this dialog doesn't update properly. Might just want to make a `branch` here instead.

![image](https://user-images.githubusercontent.com/4471575/228082681-4d6137e3-9e3c-48ad-99ad-a7d37577df34.png)

				`	"I'm sure whatever you lost wasn't worth all that much."`
					goto worthless
				`	"Well, I hope you had insurance on that ship, at least."`
					goto insurance
			label help
			`	He stops shaking you for a moment to say, "Well, you shouldn't have blown up my <npc model> while you were trying to help, weren't you? Fat lot of help that was, even if you tried to do the right thing. Oh, well..."`
				goto end

			label worthless

> 			`	He looks at you increduleously, scrunching up his face. "Not- not that much? Is that what you just said?" He pauses for a moment to look at the wall above you, looking lost, then he stares you down again with even more intensity. "That was my life's work up there, an <npc model> that had served me for 14 years, and you have the audacity to say it was all for nothing?!"`

Reviewer 4: ```suggestion
			`	He looks at you incredulously, scrunching up his face. "Not--not that much? Is that what you just said?" He pauses for a moment to look at the wall above you, looking lost, then he stares you down again with even more intensity. "That was my life's work up there, an <npc model> that had served me for 14 years, and you have the audacity to say it was all for nothing?!"`
```

			choice
				`	"Well, yeah."`
				`	"No, that's not what I meant."`
			`	He scoffs at you, and lets you go with a rough push. "I don't need your explanation, I don't think you could give me one. I just can't understand how you can walk away feeling alright that you've set back one man's life so much."`
				goto end
			
			label insurance

> 			`	He looks off to his left for just a moment with a look of loss in his eyes, then lets you go with a rough push. "Mmm, no, like a cocksure bastard I sure didn't. Ah, now that one isn't your fault, but if you had just taken your finger off the trigger for a few seconds less, I wouldn't be in this position now, would I? Bastards, you and the hijackers..."`

Reviewer 4: Ship insurance isn't really a thing. See the insurance agent news.

Author: Fair enough, I'll remove this part.

			label end

> 			`	Suddenly, he turns around and walks off towards the direction of the bar, leaving you to contemplate your actions.`

Reviewer 4: ```suggestion
			`	He lets go and walks off towards the direction of the bar, leaving you to contemplate.`
```

Author: In some of the various bits of dialogue, he's already let you go. I'll look through those again and see what I can change, because now I see that in some of them the captain doesn't let you go.

	on complete
		payment 28000
		conversation 
			`As you land, Syndicated Security have the <npc> in tow, escorted by a trio of Quicksilvers. The formation lands on one of the larger pads reserved for Syndicated Security vessels, fenced off from the rest of the spaceport.`

> 			`	The hijackers, just a raucous group of young kids, are hauled out of the <npc model> by Syndicated Security personnel towards an armored van. Multiple times the kids try to run away, but after a few minutes they're finally detained; the van ultimately drives off towards the spaceport terminal.`

Reviewer 4: ```suggestion
			`	The hijackers, just a raucous group of young kids, are hauled out of the <npc model> by Syndicated Security personnel towards an armored van. Multiple times the kids try to run away, but after a few minutes they're finally detained; the van drives off towards the spaceport terminal.`
```

	source
		government "Syndicate"
		not attributes "station"

> 		near "Alcyone" 4
> 		near "Almach" 3
> 		near "Gienah" 3

Reviewer 6: ```suggestion
		not near "Sol" 3
```

			`	Suddenly, he turns around and walks off towards the direction of the bar, leaving you to contemplate your actions.`
				decline

	on visit
		fail

	on complete
		log `Helped an unfortunate captain recover his Argosy after a bunch of delinquents hijacked it.`
		payment 48000
		conversation 
			`As you land, Syndicated Security have the <npc> in tow, escorted by a trio of Quicksilvers. The formation lands on one of the larger pads reserved for Syndicated Security vessels, fenced off from the rest of the spaceport.`
			`	The hijackers, just a raucous group of young kids, are hauled out of the <npc model> by Syndicated Security personnel towards an armored van. Multiple times the kids try to run away, but after a few minutes they're finally detained; the van drives off towards the spaceport terminal.`
			`	After the incident is wrapped up, the captain of the <npc> sends you <payment>, and a message thanking you for helping him recover his ship.`

mission "Argosy Hijacking (Declined)"
	invisible
	landing
	source
		government "Syndicate"
		not attributes "station"
		not near "Sol" 3

> 	to offer
> 		has "Argosy Hijacking (Trigger): offered"
> 	on offer
> 		fail

Reviewer 4: This prevents save-reload bugs.
```suggestion
	deadline 1
	to offer
		has "Argosy Hijacking (Trigger): offered"
	to complete
		never
```
